.gitignore

```diff
@@ -25,3 +25,5 @@ data
 log
 
 .env
+.vs
+.vscode
```

Comment:
Why ignoring .vscode? It contains essential files for project setting.

Reply:
I just switched from vs to vscode with your tip and forget to edit it.